ocs_ci/ocs/resources/pod.py

```diff
@@ -1526,9 +1526,11 @@ def run_io_and_verify_mount_point(pod_obj, bs="10M", count="950"):
 def get_pods_having_label(
     label,
     namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+    retry=0,
```

Comment:
why do you need for a retry arg here?

Reply:
In my test i use the KCS article to resize noobaa db pv. during that process some of the operators are scaled down and scaled up. I use get_pods_having_label there to fetch the pod info to create pod objects, which futher used to verify pod status. This function is used immediately after the scale up step. sometimes the pods wont even be present that quicker. Having hard coded sleep might not be logical. so i extended the available retry parameter in the get() method to get_pods_having_label.